addons/website_event/controllers/main.py

```diff
@@ -293,6 +292,9 @@ def _process_attendees_form(self, event, form_details):
                 registrations.setdefault(registration_index, dict())[field_name] = int(value) or False
                 continue
 
+            if len(key_values) != 3:
```

Comment:
What about logging something, like "Received unexpected TryingToPhishStuff registratation value" ? With maybe an explicit opt-out for recaptcha brol.

Reply:
You mean the `ValueError` while unpacking was done on purpose ? Imagine a custom code that would want to process extra (non-standard) fields in a different way, we should consider them as `TryingToPhishStuff` too ?

Reply:
Yes indeed with custo you could have logs for 'valid' stuff. Ok then, anyway this whole stuff is a bit broken by design, I suppose it is already better than nothing.